packages/create-turbo/src/transforms/types.ts

```diff
@@ -21,10 +21,20 @@ export interface TransformInput {
   opts: CreateCommandOptions;
 }
 
+export interface MetaJson {
+  maintainedByCoreTeam?: string;
+  name?: string;
+  description?: string;
+  featured?: boolean;
+  boost?: boolean;
+  template?: boolean;
+}
+
 export interface TransformResponse {
   // errors should be thrown as instances of TransformError
   result: "not-applicable" | "success";
   name: string;
+  metaJson?: MetaJson;
```

Comment:
Let's make this more generic and just call it `data`. This response type is re-used by every transform, but the metaJson is only applicable to one.